packages/flutter/lib/src/cupertino/colors.dart

```diff
@@ -611,7 +611,7 @@ abstract final class CupertinoColors {
     highContrastColor: Color.fromARGB(94, 60, 60, 67),
     darkHighContrastColor: Color.fromARGB(173, 84, 84, 88),
     elevatedColor: Color.fromARGB(73, 60, 60, 67),
-    darkElevatedColor: Color.fromARGB(153, 84, 84, 88),
+    darkElevatedColor: Color.fromARGB(153, 210, 210, 210),
```

Comment:
Can you tell me why this change is needed? Actually I don't see where this variable is used in Cupertino...